Checklist
- the common schema is pointing to main, I manually did a `git checkout main` to verify
- @rly the pynwb tests are not passing though I don't think it is on the hdmf side. 
```
FAILED tests/read_dandi/test_read_dandi.py::TestReadNWBDandisets::test_read_first_nwb_asset - SystemExit: 1
FAILED tests/unit/test_file.py::TestNoCacheSpec::test_simple - AssertionError: UserWarning not triggered
FAILED tests/validation/test_validate.py::TestValidateCLI::test_validate_file_no_cache - AssertionError: Regex didn't match: '.*UserWarning: No cached namespaces found in tests/back_compat/1\\.0\\.2_nwbfile\\.nwb\\s*warnings.warn\\(msg\\)\\s*The file tests/back_compat/1\\.0\\.2_nwbfile\\.nwb has no cached namespace information\\. Falling back to...
FAILED tests/validation/test_validate.py::TestValidateCLI::test_validate_file_no_cache_bad_ns - AssertionError: Regex didn't match: ".*UserWarning: No cached namespaces found in tests/back_compat/1\\.0\\.2_nwbfile\\.nwb\\s*warnings.warn\\(msg\\)\\s*The file tests/back_compat/1\\.0\\.2_nwbfile\\.nwb has no cached namespace information\\. Falling back to...```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # HDMF Changelog
 
-## HDMF 3.9.1 (September 28, 2023)
+## HDMF 3.9.1 (September 29, 2023)
 
 ### Enhancements
 - Updated `TermSet` to be used with `TermSetWrapper`, allowing for general use of validation for datasets and attributes. This also brings updates to `HERD` integration and updates on `write` to easily add references for wrapped datasets/attributes. @mavaylon1 [#950](https://github.com/hdmf-dev/hdmf/pull/950)

